doc(operations) : add description (#848)
bonitasoft/bonita-doc: md/operations.md

# Operations

> This page defines operations at task level and provides information on different types of operations that are available.

Reviewer: ```suggestion
This page defines the concept of Bonita operations, and provides information on different types of operations that are available.
```
